src/hooks/metamask.hook.ts

```diff
@@ -18,7 +18,7 @@ export interface MetaMaskInterface {
   requestChangeToBlockchain: (blockchain?: Blockchain) => Promise<void>;
   requestBalance: (account: string) => Promise<string | undefined>;
   sign: (address: string, message: string) => Promise<string>;
-  addContract: (address: string, svgData: string) => Promise<boolean>;
+  addContract: (asset: Asset, svgData: string, blockchain?: Blockchain) => Promise<boolean>;
```

Comment:
Maybe rename `blockchain` to `currentBlockchain`?

Reply:
fine for me. at first I named it `currentBlockchain` and then afterwards changed it to just `blockchain`. I am fine with both. will adapt it